ci/jenkins/test.sh

```diff
@@ -425,6 +433,172 @@ function deliver_antrea_windows {
     rm -f antrea-windows.tar.gz
 }
 
+function deliver_antrea_windows_containerd {
+    echo "====== Cleanup Antrea Installation ======"
+    clean_up_one_ns "antrea-test"
+    kubectl delete -f ${WORKDIR}/antrea-windows-containerd.yml --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/kube-proxy-windows-containerd.yml --ignore-not-found=true || true
+    kubectl delete daemonset antrea-agent -n kube-system --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/antrea.yml --ignore-not-found=true || true
+
+    echo "====== Building Antrea for the Following Commit ======"
+    export GO111MODULE=on
+    export GOPATH=${WORKDIR}/go
+    export GOROOT=/usr/local/go
+    export GOCACHE=${WORKSPACE}/../gocache
+    export PATH=${GOROOT}/bin:$PATH
+
+    git show --numstat
+    make clean
+    ${CLEAN_STALE_IMAGES_CONTAINERD}
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
+    DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
+
+    echo "====== Delivering Antrea to all the Nodes ======"
+    export_govc_env_var
+
+    # Enable verbose log for troubleshooting.
+    sed -i "s/--v=0/--v=4/g" build/yamls/antrea.yml build/yamls/antrea-windows-containerd.yml
+
+    cp -f build/yamls/*.yml $WORKDIR
+    docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
+
+    # Clean docker image to save disk space.
+    ${CLEAN_STALE_IMAGES}
+
+    echo "===== Pull necessary images on Control-Plane node ====="
+    harbor_images=("agnhost:2.13" "nginx:1.15-alpine")
```

Comment:
If the test images are the same on both containerd and docker, is it possible to use global variables?

Reply:
Removed.

Reply:
You still use the images inside function `deliver_antrea_windows_containerd`, so we need to maintain two sets of images for docker and containerd in this way. If we need to introduce new images, two places should be changed ( both in the function for docker and windows), and it is easier to introduce issues if any one is forgotten.

Reply:
As we discussed, we keep two functions for different images because we are running different version kubernetes testbeds. But in the future I will build another PR to simplify the two into one function.